### PR TITLE
Tezos update

### PR DESCRIPTION
**Pull request by Hactive808**

### CODE REVIEW

File: docs/rpc/tezos-rpc-documentation/getBlocksHead.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# getBlocksHead
+
+## How to Use
+
+Enclose the following TypeScript code example in \```typescript code example \``` to use the `getBlocksHead` method.
```
Comment (Hathoriel) [inline docs/rpc/tezos-rpc-documentation/getBlocksHead.md]: copy from ChatGPT?

File: docs/rpc/tezos-rpc-documentation/list of non working methods.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+getContractsBalanceAndFrozenBonds(params: GetContractBase): Promise<any>
```
Comment (Hathoriel) [inline docs/rpc/tezos-rpc-documentation/list of non working methods.txt]: This should not be here

File: docs/rpc/tezos-rpc-documentation/getBlocksHead.md (hunk 1 above)
Comment (Hactive808) [reply docs/rpc/tezos-rpc-documentation/getBlocksHead.md]: fixed

File: docs/rpc/tezos-rpc-documentation/list of non working methods.txt (hunk 2 above)
Comment (Hactive808) [reply docs/rpc/tezos-rpc-documentation/list of non working methods.txt]: fixed

**Review by Hathoriel — APPROVED** [review]

**Pull request merged**